batch2/batch/driver/instance_pool.py

```diff
@@ -353,6 +360,53 @@ def add_instance(self, instance):
             except asyncio.CancelledError:  # pylint: disable=try-except-raise
                 raise
             except Exception:
-                log.exception('instance pool control loop: caught exception')
-
+                log.exception('in control loop')
             await asyncio.sleep(15)
+
+    async def check_on_instance(self, instance):
+        try:
+            spec = await self.gservices.get_instance(instance.name)
```

Comment:
I was going to add that back when I add track the instance health status.

This can never generate very many API calls because we sleep 1s in the loop, so at most 60/s.  At the worst, we exceed or API quota and have to wait a bit.  As far as I know, there's no charge for GCE API calls, it's bundled in the instance pricing.

At some point, we should turn this into an instances.get call that returns all the instances like with did with k8s pods.  That'll be more API efficient.